tests/hooks/useComposedCssClasses.test.ts

```diff
@@ -0,0 +1,37 @@
+import { useComposedCssClasses } from '../../src/hooks/useComposedCssClasses';
+import { renderHook } from '@testing-library/react-hooks';
+
+describe('when there are no custom classes', () => {
+  const builtInCssClasses = { container: 'block p-1' };
+
+  it('is a pass through', () => {
+    const spy = jest.spyOn(require('tailwind-merge'), 'twMerge');
```

Comment:
nit: can we have a single import at the top `import * as  tailwindMerge from 'tailwind-merge'` and do `jest.spyOn(tailwindMerge, 'twMerge');`?

Reply:
added!